src/popup/router/components/AmountSend.vue

```diff
@@ -14,19 +14,20 @@
       <div class="ml-15 text-left" style="margin-right: auto;">
         <p class="label hidden">{{ $t('pages.tipPage.empty') }}</p>
         <span class="secondary-text f-14 block l-1" data-cy="amount">
-          {{ $t('pages.appVUE.aeid') }}
+          {{ isFungibleToken ? selectedToken.symbol : $t('pages.appVUE.aeid') }}
         </span>
         <span class="f-14 block l-1 amount-currency" data-cy="amount-currency">
-          {{ formatCurrency(currencyAmount) }}
+          {{ isFungibleToken ? formatCurrency(0) : formatCurrency(currencyAmount) }}
```

Comment:
If the fungible token has a market cap this will be needed - which is going to be the case with bonded curved tokens as they will be backed by AE, which has fiat value equivalent.